utilities/resource_sync/sync.py

```diff
@@ -277,73 +302,6 @@ def copy_job_script(src_s3path, dst_s3path):
             f"{tmpdir}/{src_basename}", dst_bucket, dst_key)
 
 
-def synchronize_job(job_name, mapping):
-    """Function to synchronize an AWS Glue job.
-
-    Args:
-        job_name: The name of AWS Glue job which is going to be synchronized.
-        mapping: Mapping configuration to replace the job parameter.
-
-    """
-    logger.debug(f"Synchronizing job '{job_name}'")
-    # Get DAG per job in the source account
-    res = src_glue.get_job(JobName=job_name)
-    job = res['Job']
-    logger.debug(f"GetJob API response: {json.dumps(job, indent=4, default=str)}")
-
-    # Skip jobs which do not have DAG
-    if args.skip_no_dag_jobs and 'CodeGenConfigurationNodes' not in job:
-        logger.debug(f"Skipping job '{job_name}' because the parameter '--skip-no-dag-jobs' is true and this job does not have DAG.")
-        return
-
-    # Store source job script path
-    src_job_script_s3_url = job['Command']['ScriptLocation']
-
-    # Organize job parameters
-    job = organize_job_param(job, mapping)
-
-    # Store destination job script path
-    dst_job_script_s3_url = job['Command']['ScriptLocation']
-
-    # Copy job script
-    if args.copy_job_script:
```

Comment:
This PR is removing copy_job_script capability which is required.
Can you port this to `process_job` function?